test_utils/utils/setup_fixtures.py

```diff
@@ -221,3 +221,44 @@ def create_boms(settings, only_create=None):
 		bom_doc = frappe.new_doc("BOM")
 		bom_doc.update(bom)
 		bom.save()
+
+
+def create_employees(settings, only_create=None):
+	employees = get_fixtures_data_from_file(filename="employees.json")
+	addresses = get_fixtures_data_from_file(filename="addresses.json")
+
+	for employee in employees:
+		if only_create and employee.get("employee_name") not in only_create:
+			continue
+
+		if frappe.db.exists(
+			"Employee", {"employee_name": employee.get("employee_name")}
+		):
+			continue
+
+		empl = frappe.new_doc("Employee")
+		empl.update(employee)
+		empl.save()
```

Comment:
@MyuddinKhatri says that we should make this company-sensitive. I propose this pattern:
```python
if settings.company:
     empl.company = settings.company
```